### PR TITLE
s2i docker file update.

### PR DESCRIPTION
**Pull request by mrsiano**
changes:
- using right perf repo for latest kernel.
- merge the kernel pkgs installation to 1 line.
- finally download the stac bins from the stac url.

Fix: #269

### CODE REVIEW

**Comment by mrsiano** [comment]
@vikaschoudhary16 please review

File: networking/synthetic/stac-s2i-builder-image/Dockerfile [hunk 1]
```diff
@@ -7,10 +7,10 @@ ENV container docker
 ARG github_url_stac_config
 
 RUN git clone ${github_url_stac_config}
-RUN wget http://mirror.centos.org/centos/7/updates/x86_64/Packages/kernel-`uname -r`.rpm \
-    && wget http://mirror.centos.org/centos/7/updates/x86_64/Packages/kernel-devel-`uname -r`.rpm \
-    && yum install -y kernel*.rpm
-RUN yum install --assumeyes \
+RUN echo -e "[rhel-7-latesn]\nname=rhel7-latest\nbaseurl=http://download.devel.redhat.com/rel-eng/latest-RHEL-7/compose/Server/x86_64/os/\nenabled=1\ngpgcheck=0" > /etc/yum.repos.d/rhel7.repo
```
Comment (vikaschoudhary16) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: is this intentional, `latesn` ?

File: networking/synthetic/stac-s2i-builder-image/Dockerfile [hunk 2]
```diff
@@ -44,4 +44,4 @@ RUN set -x && config_repo=`echo "${github_url_stac_config}" | rev | cut -d / -f
     && wget -q ${Onload_tar_url} \
     && onload_tar_file=`echo "$Onload_tar_url" | rev | cut -d / -f 1 | rev` \
     && rpmbuild -tb $onload_tar_file --quiet
```
Comment (vikaschoudhary16) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: missing `\` on L#46

File: networking/synthetic/stac-s2i-builder-image/Dockerfile (hunk 1 above)
Comment (mrsiano) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: It is provided by https://github.com/redhat-performance/perf-dept/blob/master/repo_files/rhel7-latest.repo

BTW, what do you think about make it configurable ?

File: networking/synthetic/stac-s2i-builder-image/Dockerfile (hunk 2 above)
Comment (mrsiano) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: will fix it

**Comment by mrsiano** [comment]
@openshift/svt

File: networking/synthetic/stac-s2i-builder-image/Dockerfile [hunk 3]
```diff
@@ -43,5 +43,5 @@ RUN set -x && config_repo=`echo "${github_url_stac_config}" | rev | cut -d / -f
     && source ${config_repo%.git*}/stac_config \
     && wget -q ${Onload_tar_url} \
     && onload_tar_file=`echo "$Onload_tar_url" | rev | cut -d / -f 1 | rev` \
-    && rpmbuild -tb $onload_tar_file --quiet
-
+    && rpmbuild -tb $onload_tar_file --quiet \
```
Comment (sjug) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: Eldad, I realize you're following the same pattern as before but can you please move to the conventional pattern for these:
`rpmbuild -tb $onload_tar_file --quiet && \`

File: networking/synthetic/stac-s2i-builder-image/Dockerfile (hunk 1 above)
Comment (sjug) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: I'm not sure that is a public repo, considering you linked to a private git repo

File: networking/synthetic/stac-s2i-builder-image/Dockerfile (hunk 3 above)
Comment (mrsiano) [inline networking/synthetic/stac-s2i-builder-image/Dockerfile]: sure thing.

**Comment by mrsiano** [comment]
@sjug yep it worked

**Comment by mrsiano** [comment]
@vikaschoudhary16 regarding your last findings about stac, would you like to apply the changes via this PR or new one?

**Comment by mrsiano** [comment]
@vikaschoudhary16 also I guess it depends eachother,
what we can do next is, base on your findings we can merge this one into yours

**Comment by mffiedler** [comment]
@mrsiano Do you plan to update this based on the comments?  or should I close it out?

**Comment by mrsiano** [comment]
@mffiedler yep

File: networking/synthetic/stac-s2i-builder-image/Dockerfile (hunk 1 above)
Comment (mrsiano) [reply networking/synthetic/stac-s2i-builder-image/Dockerfile]: right, so what I gonna do is, using our private repo and place it as arg there and use it for that file

**Comment by mrsiano** [comment]
@mffiedler @sjug @vikaschoudhary16 please review

**Comment by mffiedler** [comment]
Since there was no objection to the comment from 13 September, I am closing this inactive PR.

**Pull request closed without merging**